app/views/users/show.html.erb

```diff
@@ -2,7 +2,14 @@
   <div class="user-show" id="welcome">
     <h2>Welcome, <%= @user.username %></h2>
     <h3><%= "#{@user.username}'s Games" %></h3>
+    <% if @user.game.id == 1 %>
```

Comment:
Don't really need to use interpolation in erb.  Should be able to get away with 
<h3><%= @user.username>'s Games %> </h3>